### PR TITLE
KOGITO-2826: Fixed grey space near load more

### PR DESCRIPTION
**Pull request by Sara4994**
Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Comment by Sara4994** [comment]
jenkins please retest this

**Review by AjayJagan — APPROVED** [review]
:+1:

**Comment by Sara4994** [comment]
jenkins please retest this

File: ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx [hunk 1]
```diff
@@ -35,19 +35,19 @@ const LoadMore: React.FC<IOwnProps & OUIAProps> = ({
   };
 
   const load10More = () => {
-    setLoadMoreClicked(true);
+    setLoadMoreClicked && setLoadMoreClicked(true);
```
Comment (nmirasch) [inline ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx]: @Sara4994 if we are doing this for every loadXMore function, maybe we can add this into  loadMore()

Comment (Sara4994) [reply ui-packages/packages/common/src/components/Atoms/LoadMore/LoadMore.tsx]: Thanks @nmirasch this is done :+1:

**Comment by nmirasch** [comment]
jenkins please retest this

**Comment by nmirasch** [comment]
jenkins please retest this

**Comment by Sara4994** [comment]
jenkins please retest this

**Review by nmirasch — APPROVED** [review]

**Review by jstastny-cz — APPROVED** [review]

**Review by cristianonicolai — APPROVED** [review]

**Pull request merged**